.gitignore

```diff
@@ -1,3 +1,5 @@
 gyp/test
 node_modules
 test/.node-gyp
+.ncu
+package-lock.json
```

Comment:
I would actually commit the package-lock.json
see: https://stackoverflow.com/a/44210813

Reply:
Why is this ignored?

Reply:
> Why is this ignored?

I too wonder why is this ignored... cc @refack ...